export.py

```diff
@@ -2949,47 +2947,55 @@ def export_display(ri, rpass, scene):
-                    # looks like someone didn't set an lpe string
-                    if source == '':
-                        continue
-                elif source == 'custom_aov_string':
-                    source = aov.custom_aov_string
-                    source_type = aov.custom_aov_type
-                    if source == '':
-                        continue
-                elif source == 'built_in_aov':
-                    source_type, source = aov.aov_channel_type.split()
-                elif source in ['z', 'u', 'v', 'id']:
-                    source_type = "float"
-                elif source is 'Nn':
-                    source_type = 'normal'
-                elif source is 'dPdtime':
-                    source_type = 'vector'
-                else:
-                    # light groups need to be surrounded with '' in lpes
-                    G_string = "'%s'" % rm_rl.object_group if rm_rl.object_group != '' else ""
-                    LG_string = "'%s'" % rm_rl.light_group if rm_rl.light_group != '' else ""
+
+                name_map = [
+                    ("directDiffuseLobe", "diffuse11"),
+                    ("subsurfaceLobe", "diffuse12"),
+                    ("Subsurface", "diffuse13"),
+                    ("transmissiveSingleScatterLobe", "diffuse14"),
+                    ("Caustics", "diffuse15"),
+                    ("Albedo", "diffuse16"),
+                    ("Diffuse", "diffuse17"),
+                    ("IndirectDiffuse", "indirectdiffuse11"),
+                    ("indirectDiffuseLobe", "indirectdiffuse12"),
+                    ("directSpecularPrimaryLobe", "specular11"),
+                    ("directSpecularRoughLobe", "specular12"),
+                    ("directSpecularClearcoatLobe", "specular13"),
+                    ("directSpecularIridescenceLobe", "specular14"),
+                    ("directSpecularFuzzLobe", "specular15"),
+                    ("directSpecularGlassLobe", "specular16"),
+                    ("transmissiveGlassLobe", "specular17"),
+                    ("Reflection", "specular18"),
+                    ("Refraction", "specular19"),
+                    ("Specular", "specular20"),
+                    ("indirectSpecularPrimaryLobe", "indirectspecular11"),
+                    ("indirectSpecularRoughLobe", "indirectspecular12"),
+                    ("IndirectSpecular", "indirectspecular13"),
+                    ("indirectSpecularClearcoatLobe", "indirectspecular14"),
+                    ("indirectSpecularIridescenceLobe", "indirectspecular15"),
+                    ("indirectSpecularFuzzLobe", "indirectspecular16"),
+                    ("indirectSpecularGlassLobe", "indirectspecular17"),
+                    ("Shadows", "emission10"),
+                    ("Emission", "emission11")
+                ]
+
+                # Remaps any color lpe channel names to a denoise friendly one
+                for name, channel in name_map:
```

Comment:
Then here do:
if aov_name in name_map.keys():
     aov.channel_name = name_map[aov_name]